rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -760,7 +742,7 @@ stream {
             proxy_set_header       X-Request-ID       $req_id;
             proxy_set_header       Host               $best_http_host;
 
-            set $proxy_upstream_name {{ $upstreamName }};
+            set $proxy_upstream_name {{ $upstreamName | quote }};
```

Comment:
was there any edge case causing issues? why are you adding `quote` now?

Reply:
consistency.